CONTRIBUTING.md

```diff
@@ -22,6 +22,11 @@ You can see the `npm` scripts available inside the `package.json` file and use t
 * Commits should follow the [Conventional Commit](https://www.conventionalcommits.org/) standard: `<type>(<scope>): <subject>` (e.g. `feat(core): Add new feature`);
 * At commit time, `yarn test` command script is automatically run. To skip this step, commit with `-n` flag.
 
+##### Pull Requests
```

Comment:
Does the PR template file include a notice about "making sure you've read the contributing guidelines, specifically for opening PR?